packages/browser/docs/api/Makefile

```diff
@@ -33,36 +33,40 @@ migrate: clean
 
    # Using https (not ssh) to clone
 	git clone https://github.com/inrupt/docs-assets.git build/docs-assets
-   
+
+
 	# Copying to SOURCECOPYDIR instead of copying source dir to BUILDDIR
    # in case someone forgets to backslash after build/
 
-	cp -R $(SOURCEDIR)/index.rst $(SOURCECOPYDIR)
 	cp -R $(SOURCEDIR)/api/* $(SOURCECOPYDIR)
 
-	# Temporary workaround for the functions being in index.md (which is the README.md file) instead of a separate module
+	# Temporary workaround for the functions being in generated index.rst (which is the README.md file) instead of a separate module
```

Comment:
```suggestion
	# Temporary workaround for the functions being in the generated index.rst (which is the README.md file) instead of a separate module
```

Reply:
Thanks.  Will also add a patch to include a new line in the other files.